config/initializers/address_lookup.rb

```diff
@@ -0,0 +1,8 @@
+EA::AddressLookup.configure do |config|
+  config.address_facade_server  = ENV["ADDRESS_FACADE_TEST_SERVER"]
```

Comment:
I think we should only be using a test server, when in the test environment - or perhaps in develop in some circumstances. So the environment variable should be `ENV["ADDRESS_FACADE_SERVER"]`